content/applications/websites/elearning.rst

```diff
@@ -0,0 +1,258 @@
+of the :doc:`Surveys <../marketing/surveys/overview/create>` app.
+
+.. _elearning/description:
+
+Description tab
+~~~~~~~~~~~~~~~
+
+Add a short description of your course. It appears under your course title on your website.
+
+.. image:: elearning/course-description.png
+   :align: center
+   :alt: Add a description to your course.
+
+.. _elearning/options:
+
+Options tab
+~~~~~~~~~~~
+
+This tab is divided into different sections: :ref:`Course <elearning/course>`,  :ref:`Communication
+<elearning/communication>`, :ref:`Access rights <elearning/access-rights>`, :ref:`Display
+<elearning/display>`.
+
+.. image:: elearning/options-tab.png
+   :align: center
+   :alt: Overview of the Options tab
+
+.. _elearning/course:
+
+Course
+******
+
+Set up who is :guilabel:`Responsible` for your course, and select a :guilabel:`Website` if you want
+your course to be restricted to this website only.
+
+.. _elearning/communication:
+
+Communication
+*************
+
+- :guilabel:`Allow Reviews`: tick the box to allow attendees to like and comment your content and to
+  submit reviews on your course;
+- :guilabel:`Forum`: add a dedicated forum to your course (only possible if **Forum** is installed);
+- :guilabel:`New Content Notification`: select an email template that is sent to your attendees when
+  you upload new content. Tick the green arrow to update the email template;
+- :guilabel:`Completion Notification`: select an email template that is sent to your attendees once
+  they reach the end of your course. Tick the green arrow to update the email template.
+
+.. _elearning/access-rights:
+
+Access rights
+*************
+
+- :guilabel:`Show course to`: define who can access your course and their content between
+  :guilabel:`Everyone`, :guilabel:`Signed In` or :guilabel:`Course Attendees`;
+- :guilabel:`Enroll Policy`: define how people enroll to your course. Select:
+
+   - :guilabel:`Open`: your course is available to anyone;
+   - :guilabel:`On Invitation`: if selected, set up the :guilabel:`Enroll message` explaining the
+     enroll process to subscribe to the course. This message appears on your website under the
+     course title;
+   - :guilabel:`On Payment`: the :guilabel:`Paid Courses` features must be enabled in your settings
+     to get this option. If you select :guilabel:`On Payment`, you must add a :guilabel:`Product`
+     for your course. Select an existing one or create a new one. It can be updated by clicking on
+     the green arrow that redirects you to the :doc:`../sales/sales` app to finalize the
+     configuration.
+
+.. _elearning/display:
+
+Display
+*******
+
+- :guilabel:`Training`: the course content appears as a training program, and the courses must be
+  taken in the proposed order.
+- :guilabel:`Documentation`: the content is available any order. If you chose this option, you can
+  select the :guilabel:`Featured Content` on the course homepage.
```

Comment:
- :guilabel:`Documentation`: the content is available in any order. If you choose this option, you can
  select the :guilabel:`Featured Content` on the course homepage.